python/ray/rllib/a3c/shared_model.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+import distutils.version
+from ray.rllib.a3c.policy import (
+    categorical_sample, linear,
+    normalized_columns_initializer, Policy)
+
+from ray.rllib.models.catalog import ModelCatalog
+
+class SharedModel(Policy):
+
+    def __init__(self, ob_space, ac_space, **kwargs):
+        super(SharedModel, self).__init__(ob_space, ac_space, **kwargs)
```

Comment:
The LSTM setup should be placed alongside the rest of the shared models. The LSTM wrapper class for A3C will probably be a subclass of this SharedModel class.